### PR TITLE
Update tuneup to 2.10.1

### PR DESCRIPTION
**Pull request by SentinelWarren**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/tuneup.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+  version '2.10.1'
+  sha256 '4332684d776c7e3114bc19357360b51fba69c77b8c4c641ed3a518ab9939fe86'
 
-  # dvk2ozaytrec6.cloudfront.net was verified as official when first introduced to the cask
-  url "https://dvk2ozaytrec6.cloudfront.net/mac4/Sparkle/TuneUp-#{version}.dmg"
-  appcast 'https://dvk2ozaytrec6.cloudfront.net/mac4/appcast.xml',
-          checkpoint: '64150d576a3291c6755d79e68b2e52e5d2fd71bee8e7627ac77bf57aca4f6f55'
+  # d2m5os1cpndbxq.cloudfront.net was verified as official when first introduced to the cask
+  url 'http://d2m5os1cpndbxq.cloudfront.net/mac4/Sparkle/TuneUp-Installer.dmg'
+  appcast 'https://d2m5os1cpndbxq.cloudfront.net/mac4/appcast.xml',
```
Comment (miccal) [inline Casks/tuneup.rb]: This `appcast` does not seem to be valid - how did you find it?

**Comment by miccal** [comment]
@colindunn I am not sure what process you use to find `appcast`'s - any chance you can have a look at this one?

**Comment by colindunn** [comment]
Hey miccal@, I do it manually by viewing the network traffic etc. I can't find one looking through the site, nor can I find a versioned download link.

**Comment by miccal** [comment]
Thanks @colindunn.

**Comment by miccal** [comment]
I also notice that this has `installer manual:` but no `uninstall`. Considering also that this has `malware`, I'm inclined to remove this from Cask.

@vitorgalvao thoughts?

**Comment by vitorgalvao** [comment]
> I also notice that this has `installer manual:` but no `uninstall`. Considering also that this has malware, I'm inclined to remove this from Cask.

> @vitorgalvao thoughts?

Agreed!

**Pull request closed without merging**